static/js/custom-render.js

```diff
@@ -21,4 +21,30 @@ Object.assign(render, {
       ? `<img src="${data.location}" style="max-height:150px;" loading="lazy">`
       : null
   },
+  finished: function render(data, type, full, meta, fieldOptions) {
+    if (data == null) return null_column();
+    if (Array.isArray(data) && data.length == 0) return empty_column();
+    data = Array.isArray(data) ? data : [data].map((d) => d === true);
+    if (type != "display") return data.join(",");
```

Comment:
☝️ what's the use case for when `type != "display"` and what is the visual effect of `return data.join(",")`?

Reply:
🤷 Directly copied from https://github.com/WGBH-MLA/starlette-admin/blob/855a71ccf7b7a4fa6ccd3026a9bf593eccb18d7c/starlette_admin/statics/js/render.js#L42-L54

I think it's for